src/criteria_assessment/query_thresholds.jl

```diff
@@ -201,7 +201,7 @@ applied to a set of criteria.
 
 # Notes
 - Zero values indicate locations to mask **out**.
-- Ones indicate locations to **keep**.
+- One values indicate locations to **keep**.
```

Comment:
How about "Zeros" and "Ones"?

Removes the need to say "values"

e.g., "Zeros indicate locations..."